internal/pikamon/config.go

```diff
@@ -20,6 +20,12 @@ type Config struct {
 		DebounceWindow time.Duration
```

Comment:
Now that this has a more well defined usage as how long a pokemon is spawned for, lets make this the something more like minimum existance duration.

Reply:
I have changed this to `minimumSpawnDuration`